ts/components/ui/cards/payment/PaymentCardBig.tsx

```diff
@@ -0,0 +1,212 @@
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import Placeholder from "rn-placeholder";
+import { WithTestID } from "../../../../types/WithTestID";
+import { formatDateAsLocal } from "../../../../utils/dates";
+import { IOLogoPaymentType, LogoPayment } from "../../../core/logos";
+import { VSpacer } from "../../../core/spacer/Spacer";
+import { LabelSmall } from "../../../core/typography/LabelSmall";
+import { NewH6 } from "../../../core/typography/NewH6";
+import { IOColors } from "../../../core/variables/IOColors";
+import { IOBannerRadius } from "../../../core/variables/IOShapes";
+import { IOStyles } from "../../../core/variables/IOStyles";
+import { LogoPaymentOrDefaultIcon } from "../../utils/baseComponents/LogoPaymentOrDefaultIcon";
+import I18n from "../../../../i18n";
+
+const styles = StyleSheet.create({
+  cardContainer: {
+    flex: 1,
+    alignItems: "flex-start",
+    alignContent: "space-between",
+    justifyContent: "space-between",
+    height: 207,
+    borderRadius: IOBannerRadius,
```

Comment:
```suggestion
    borderRadius: 16,
```

Reply:
addressed in [2dee23f](https://github.com/pagopa/io-app/pull/4759/commits/2dee23ff30ceeefd9590eb9681760aa500302c1b)